docs/user-guide/dream/dream-data-reduction.ipynb

```diff
@@ -53,6 +53,8 @@
     "    # Mask in time-of-flight to crop to valid range\n",
     "    TofMask: lambda x: (x < sc.scalar(0.0, unit=\"ns\"))\n",
     "    | (x > sc.scalar(86e6, unit=\"ns\")),\n",
+    "    TwoThetaMask: None,\n",
+    "    WavelengthMask: None,\n",
```

Comment:
Can we have default params so users don't have to specify everything? I thought that was the plan anyway?

Reply:
Yes, but as far as I could tell ESSdiffraction is not implementing C.5 yet (see https://github.com/scipp/essreduce/blob/main/docs/user-guide/reduction-workflow-guidelines.md#c5-use-a-fixed-pattern-for-creating-manipulating-and-running-workflows), so this is unrelated to this PR?